### PR TITLE
Update fpga autorun readme

### PR DESCRIPTION
**Pull request by jkinsky**
# Existing Sample Changes
## Description

Restructured the readme to match the new template. Updated formatting. Updated some branding. Corrected some spelling and grammar. Moved images into /images folder to help clean up top-level folder.

Fixes Issue# 

## External Dependencies

N/A

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [x] Documentation

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Command Line
- [x] VSCode

### CODE REVIEW

**Comment by jkinsky** [comment]
Adding @jimmytwei for notices.

**Comment by yuguen-intel** [comment]
Interestingly I made the same kind of changes to put images in a specific folder for the reference designs.
I tried to check if there was a commonly used name for this folder in other GitHub repos and I found multiple using the name "assets" rather than "images". 
So I used that (here https://github.com/oneapi-src/oneAPI-samples/pull/1208) 
We should keep this consistent, would you be happy about using the "assets" name?

File: DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/autorun/README.md [hunk 1]
```diff
@@ -1,215 +1,235 @@
...
-    ```
-    cmake ..
-   ```
-
-   Alternatively, to compile for the Intel&reg; FPGA PAC D5005 (with Intel Stratix&reg; 10 SX), run `cmake` using the command:
-
+   For **Intel® FPGA PAC D5005 (with Intel Stratix® 10 SX)**, enter the following:
    ```
-   cmake .. -DFPGA_DEVICE=intel_s10sx_pac:pac_s10
+   cmake .. -DFPGA_BOARD=intel_s10sx_pac:pac_s10
```
Comment (yuguen-intel) [inline DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/autorun/README.md:91]:
```suggestion
   cmake .. -DFPGA_DEVICE=intel_s10sx_pac:pac_s10
```
FPGA_DEVICE here is the expected name to be used

File: DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/autorun/README.md [hunk 2]
```diff
@@ -1,215 +1,235 @@
...
-    ```
-    cmake -G "NMake Makefiles" ..
-   ```
-
-   Alternatively, to compile for the Intel&reg; FPGA PAC D5005 (with Intel Stratix&reg; 10 SX), run `cmake` using the command:
-
+   To compile for the **Intel® FPGA PAC D5005 (with Intel Stratix® 10 SX)**, enter the following:
    ```
-   cmake -G "NMake Makefiles" .. -DFPGA_DEVICE=intel_s10sx_pac:pac_s10
+   cmake -G "NMake Makefiles" .. -DFPGA_BOARD=intel_s10sx_pac:pac_s10
```
Comment (yuguen-intel) [inline DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/autorun/README.md:134]: Same remark

Comment (yuguen-intel) [inline DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/autorun/README.md:134]:
```suggestion
   cmake -G "NMake Makefiles" .. -DFPGA_DEVICE=intel_s10sx_pac:pac_s10
```

**Review by yuguen-intel — CHANGES_REQUESTED** [review]

**Comment by jkinsky** [comment]
> Interestingly I made the same kind of changes to put images in a specific folder for the reference designs. I tried to check if there was a commonly used name for this folder in other GitHub repos and I found multiple using the name "assets" rather than "images". So I used that (here #1208) We should keep this consistent, would you be happy about using the "assets" name?

I think it makes little difference what the directory is named. You can find instances of /images plus others sub-directories. I think assets might be too generic, but I really don't care that much. The main thing I hoped to accomplish was to remove visual clutters from the top-level directory.

**Comment by yuguen-intel** [comment]
I completely agree that is visually makes things simpler, which is a good thing, I just want all the FPGA samples to be consistent :)

**Comment by jkinsky** [comment]
Amended a commit. I might have amended the wrong commit. I'm confused over what should be a relatively simple change. Sorry I started the process.

**Review by ishaghosh27 — APPROVED** [review]

**Review by yuguen-intel — APPROVED** [review]

**Pull request merged**